package.json

```diff
@@ -62,6 +62,7 @@
     "react-native-immersive": "1.0.0",
     "react-native-keep-awake": "2.0.5",
     "react-native-locale-detector": "1.0.1",
+    "react-native-progress": "3.4.0",
```

Comment:
Can you please use react-native's `ActivityIndicator` instead?

Reply:
Sure, I didn't know it existed! Less dependencies, more better!